.gitattributes

```diff
@@ -0,0 +1,40 @@
+# Handle line endings automatically for files detected as text
```

Comment:
We should add necessary attributes here as needed. Please remove extra attributes.

Reply:
These attributes are general purpose attributes and suitable for every web application